pyproject.toml

```diff
@@ -31,3 +31,42 @@ spelling-dict = ['en_US']
 disable = "all"
 enable = "C0402, C0403" 
 spelling-private-dict-file = "wordlist.txt"
+
+
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "QuCoNot"
+version = "0.01"
+authors = [
+  { name="Adam Glos", email="aglos@iitis.pl" },
+]
+description = 'QuCoNot is a Python library for all your MCT needs.'

```

Comment:
I would write more about it. First of all MCT may have different meaning so I would write its full form. Secondly - there should be quantum context in the description

Reply:
this is placeholder text. The description will be finalised once documentation takes shape.

Reply:
OK. Just make issue for this